src/components/DecimalValue/index.js

```diff
@@ -21,17 +21,15 @@ export const decimalToText = (value, decimals = 2) => {
   return decimalValue.toDP(decimals, 1).toString()
 }
 
-const DecimalValue = ({ value, decimals = 2, className }) => {
+const DecimalValue = ({ value, decimals = 4, className }) => {
   const text = decimalToText(value, decimals)
   return <span className={className}>{text}</span>
 }
 
 // I don't use PropTypes.instanceOf because Decimal can be cloned with different default properties
 // and instanceOf doesn't deal with that situation. In fact, Decimal.clone is used in gnosis.js
 const decimalJsTest = (props, propName, componentName) => {
```

Comment:
Can you provide me with an example? And where and why do we use this function?

Reply:
`new Decimal()` and `Decimal.clone` don't work with `PropTypes.instanceOf` for some reason. I can't remember where it was, but instanceOf never worked, so a custom proptype-test was put in place.